src/esp/gfx/LightSetup.cpp

```diff
@@ -28,7 +28,7 @@ Magnum::Vector3 getLightPositionRelativeToCamera(
       return light.position;
   }
 
-  CORRADE_ASSERT_UNREACHABLE();
+  CORRADE_INTERNAL_ASSERT_UNREACHABLE();
   return {};
```

Comment:
Since you're touching this anyway, the return statement isn't needed here -- that's actually why this unreachable macro exists :)